Automattic/wp-calypso
# Add google domains explainer copy to the payments method step

Fixes https://github.com/Automattic/dotcom-forge/issues/3269

## Proposed Changes

* Add new copy to the checkout payment method step for google domain transfers to explain why we're charging them $0 but still need their credit card info.
* Includes another spot to highlight that we're paying for this.

## Testing Instructions

- http://calypso.localhost:3000/setup/google-transfer/domains
- Once on the checkout step to set payment details there should be new copy below the payment options header
- The copy should only display if you have google domains in the cart
- If you don't have a google domain, sandbox pub-api and make init_registered_with_google_domains return true.

Before

![Screenshot 2023-07-31 at 14-34-27 Checkout — WordPress com](https://github.com/Automattic/wp-calypso/assets/811776/8f096e89-a22e-4f4b-b66e-1e2e0cd93125)

After

![Screenshot 2023-07-31 at 15-03-41 Checkout — WordPress com](https://github.com/Automattic/wp-calypso/assets/811776/c8b3c51c-4c11-4524-9522-9fa1b0966c71)
